README.md

```diff
@@ -29,15 +29,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+> `map` returns values and will return a new array vs. `forEach` doesn't return anything, it calls a provided function on each element in the array
 2. What is the difference between a function and a method?
-
+> They are both functions, a method is property of a function.
 3. What is closure?
-
+> A function in which the body  has access to variables declared outside of the function
```

Comment:
Closure is more accurately a function nested in an object that is called from outside of the whole structure. 

Or a function within a function. 

It's definitely a complicated concept, but gets easier with time. Check out this very good article on closure:

https://medium.com/dailyjs/i-never-understood-javascript-closures-9663703368e8